Jenkinsfile

```diff
@@ -0,0 +1,95 @@
+pipeline {
+  agent any
+  stages {
+    stage('Docker: Network') {
+      steps {
+        sh "docker network create ${BUILD_TAG}"
+      }
+    }
+    stage('Dependencies') {
+      parallel {
+        stage('Composer') {
+          agent {
+            dockerfile {
```

Comment:
I wouldn't include docker container configurations for a library skeleton. That is more something for a (micro)service skeleton. Public images can be used instead.